test/index.js

```diff
@@ -56,6 +76,29 @@ testEncodeDecode([])
 testEncodeDecode({})
 testEncodeDecode([1, 2, 3, 4, 5, 6, 7, 8, 9])
 testEncodeDecode('hello')
+testEncodeDecode(
+  '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_' +
+    '123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_123456789_'
+)
```

Comment:
Here we are testing that `bipf` can handle strings longer than 2048 (this is actually a big deal because `bipf-native` for some time could only handle small strings). Each line above has exactly 100 characters, and there are 21 lines in total.